test/sanity/issue6143-mac-setProgressBar-crash/index.html

```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="utf-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <title>issue6143-mac-setProgressBar-crash</title>
+</head>
+<body>
+  <h3 id="result">success</h3>
+  <script>
+    chrome.downloads.download({ url: "http://releases.ubuntu.com/xenial/ubuntu-16.04.3-desktop-amd64.iso" });
```

Comment:
I think this could lead to full disk on the test machine?

Reply:
Updated the link to https://github.com/nwjs/nw.js/archive/nw-v0.27.4.tar.gz

Reply:
given enough time the disk should be still full. The correct way is to find where will the download be and remove them after the test is done.